src/app/loginComponents/accounts/external/accounts.component.ts

```diff
@@ -28,23 +28,29 @@ import { UserQuery } from '../../../shared/user/user.query';
 import { UserService } from '../../../shared/user/user.service';
 import { TokenUser } from './../../../shared/swagger/model/tokenUser';
 import { AccountsService } from './accounts.service';
+import { User } from '../../../shared/swagger/model/user';
+import { ViewEncapsulation } from '@angular/core';
 
 @Component({
   selector: 'app-accounts-external',
   templateUrl: './accounts.component.html',
-  styleUrls: ['./accounts.component.css'],
+  styleUrls: ['./accounts.component.scss'],
+  encapsulation: ViewEncapsulation.None,
 })
 export class AccountsExternalComponent implements OnInit, OnDestroy {
   public dsServerURI: any;
   public orcidId$: Observable<string>;
   public TokenSource = TokenSource;
+  public user: User;
   Dockstore = Dockstore;
   // TODO: Uncomment section when GitLab is enabled
   accountsInfo: Array<any> = [
     {
       name: 'GitHub',
       source: TokenSource.GITHUB,
       bold: 'One of GitHub or Google is required.',
+      control: 'Source Control',
```

Comment:
I don't think you're using the text itself, you're just checking if the text is empty or not (`*ngIf="row.control"` -- unless I missed one). If so, you could just make these booleans.

Also this one doesn't have `docker`, but the other ones do. It's works because because of JavaScript falsiness, but for symmetry purposes... A type would have caught it (see previous comment) :)

Also, I probably would have created a single property with an enum, but what you have works.

Reply:
Thank you! I added `docker` and just changed it to boolean for now. :)